src/components/MediaCollection/createHeartbeatTicker.js

```diff
@@ -0,0 +1,27 @@
+export default ({ config, trackMediaEvent, playerCache }) => {
```

Comment:
Are these 3 concepts the same idea: `heartbeat`, `ticket` and `playhead`? If yes, I suggest we find a way to just use one of these names for it.